packages/xod-project/src/constants.js

```diff
@@ -24,6 +24,13 @@ export const PIN_TYPE = {
   T3: 't3',
 };
 
+/**
+ * Enumeration of custom types with literals
+ */
+export const CUSTOM_TYPE = {
```

Comment:
I'm not sure if this should be called just `CUSTOM_TYPE`. For example, `isBindableCustomType`'s implementation looks a bit weird because of it.

Reply:
+1
I'll replace it with a proper (but probably longer 😞) name.